scripts/templates/EmptyTestsCommon.mustache

```diff
@@ -0,0 +1,18 @@
+/** Jest test setup file. */
+
+const { setIconOptions } = require('office-ui-fabric-react/lib/Styling');
+const { configure } = require('enzyme');
+const Adapter = require('enzyme-adapter-react-16');
+
+// Suppress icon warnings.
+setIconOptions({
+  disableWarnings: true
+});
+
+// Mock requestAnimationFrame for React 16+.
+global.requestAnimationFrame = callback => {
+  setTimeout(callback, 0);
+};
+
+// Configure enzyme.
+configure({ adapter: new Adapter() });
```

Comment:
Is this file an exact dupe of `EmptyTests.mustache` above?

Reply:
Yes it is. With the current logic, the step name that writes the file needs to match the name of the mustache template file. The content of this file are written to both config/tests.js and src/common/tests.js, so there needed to be two separate steps for this.